docker-compose.yml

```diff
@@ -85,15 +80,23 @@ services:
       timeout: 5s
       retries: 5
 
-
-# filescan:
-#   image: clamav/clamav:latest
-#   container_name: filescan
-#   ports:
-#     - "3310:3310" # Expose the ClamAV port if needed for pyClamd
-#   restart: always
-#   volumes:
-#     - clamav_db:/var/lib/clamav # Persist the virus definition database
-
-# volumes:
-#  clamav_db:
+  filescan:
```

Comment:
This is a very, very, very good start!

If we are pulling the clamav images here, do we need to also add clamav modules to requirement files?